queries/roads.jinja2

```diff
@@ -68,6 +68,14 @@ SELECT
     leisure,
     sport,
     man_made,
+{% if zoom >= 11 %}
```

Comment:
Since we don't have minor roads at zoom 11, let's make this >= 12 for this release. Otherwise it looks broken at zoom 11.